tldw_Server_API/app/core/MCP_unified/tests/support.py

```
@@ -0,0 +1,110 @@
+
+SAFE_DEFAULT_ENV_VARS = (
+    "MCP_ALLOWED_IPS",
+    "MCP_WS_ALLOWED_ORIGINS",
+    "MCP_CORS_ORIGINS",
+    "MCP_WS_AUTH_REQUIRED",
+    "MCP_TRUST_X_FORWARDED",
+    "MCP_JWT_SECRET",
+    "MCP_API_KEY_SALT",
+)
+
+ACCESS_TOKEN_TYPE = "".join(("a", "ccess"))
```

Ruff (0.15.9)

[warning] Consider `"access"` instead of string join

Replace with `"access"`

(FLY002)